packages/template-retail-react-app/app/ssr.js

```diff
@@ -36,68 +35,8 @@ const options = {
 const runtime = getRuntime()
 
 const {handler} = runtime.createHandler(options, (app) => {
-    const getRuntimeEnv = () => {
-        if (process.env.NODE_ENV !== 'production') return process.env.NODE_ENV ?? 'development'
-        const origin = getAppOrigin()
-        // mobify-storefront-staging sites have NODE_ENV set to production, but for the purposes
-        // of CSP we consider the sites to be staging.
-        return origin.endsWith('.mobify-storefront-staging.com') ? 'staging' : 'production'
-    }
-
-    // This is a temporary solution while we work on Storefront Preview - a full solution will
-    // prevent required CSP headers from being modified.
-    const getCSP = () => {
-        const trustedMap = {
-            development: [
-                'localhost:*',
-                '*.commercecloud.salesforce.com',
-                '*.demandware.net',
-                '*.mobify-staging.com',
-                '*.mobify-storefront-staging.com',
-                '*.mobify-storefront.com',
-                'runtime.commercecloud.com'
-            ],
-            staging: [
-                '*.demandware.net',
-                '*.mobify-staging.com',
-                '*.mobify-storefront-staging.com',
-                '*.mobify-storefront.com',
-                '*.commercecloud.salesforce.com',
-                'runtime.commercecloud.com'
-            ],
-            production: [
-                '*.demandware.com',
-                '*.mobify.com',
-                '*.mobify-storefront.com',
-                '*.commercecloud.salesforce.com',
-                'runtime.commercecloud.com'
-            ]
-        }
-
-        const env = getRuntimeEnv()
-        const trusted = ["'self'", ...(trustedMap[env] ? trustedMap[env] : [])]
-
-        return {
-            'connect-src': ['api.cquotient.com', ...trusted],
-            'frame-ancestors': [...trusted],
-            'img-src': ['data:', ...trusted],
-            'script-src': ["'unsafe-eval'", 'storage.googleapis.com', ...trusted],
-
-            // Do not upgrade insecure requests for local development
-            'upgrade-insecure-requests': isRemote() ? [] : null
-        }
-    }
-
     // Set HTTP security headers
-    app.use(
-        helmet({
-            contentSecurityPolicy: {
-                useDefaults: true,
-                directives: getCSP()
-            },
-            hsts: isRemote()
-        })
-    )
+    app.use(helmet({hsts: isRemote()}))
```

Comment:
A bonus point would be if the `hsts` header is disabled in the dev server, so user land is super clean.

I forgot the reason why this needs to be disabled on local dev server...

Reply:
Dev server runs on `http`; upgrading to `https` would break it.

Reply:
yea, i mean that we can conditionally set the `hsts` header in the SDK similarly to how you have done for the CSPs and developers will have 1 less line of code to worry about :)